### PR TITLE
fix go tool installation

### PR DESCRIPTION
**Pull request by kmrmt**
Signed-off-by: Kosuke Morimoto <ksk@vdaas.org>

### Description:

SSIA

### Environment:

- Go Version: 1.18.1
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.3

### Types of changes:

- [X] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: Makefile.d/test.mk [hunk 1]
```diff
@@ -92,12 +92,12 @@ coverage:
 .PHONY: tparse/install
 ## install tparse
 tparse/install:
-	go get -u github.com/mfridman/tparse
+	go install github.com/mfridman/tparse@latest
```
Comment (kpango) [inline Makefile.d/test.mk:95]:
```suggestion
	$(call go-get, github.com/mfridman/tparse)
```

File: Makefile.d/test.mk [hunk 2]
```diff
@@ -92,12 +92,12 @@ coverage:
...
 
 .PHONY: gotests/install
 ## install gotests
 gotests/install:
-	go get -u github.com/cweill/gotests/...
+	go install github.com/cweill/gotests/gotests@latest
```
Comment (kpango) [inline Makefile.d/test.mk:100]:
```suggestion
	$(call go-get, github.com/cweill/gotests/gotests)
```

**Review by kpango — CHANGES_REQUESTED** [review]

**Comment by kpango** [comment]
after suggestion commit.
could you please replace the Makefile function (go-get) name to go-install?

**Review by hlts2 — APPROVED** [review]
LGTM

File: Makefile.d/functions.mk [hunk 3]
```diff
@@ -20,7 +20,7 @@ blue   = printf "\x1b[34m\#\# %s\x1b[0m\n" $1
 pink   = printf "\x1b[35m\#\# %s\x1b[0m\n" $1
 cyan   = printf "\x1b[36m\#\# %s\x1b[0m\n" $1
 
-define go-get
+define go-install
```
Comment (kpango) [inline Makefile.d/functions.mk:23]: for this change you should also edit proto.mk

**Review by kpango — APPROVED** [review]
LGTM

**Review by hlts2 — APPROVED** [review]
LGTM

**Pull request merged**